src/app/app.component.ts

```diff
@@ -1,10 +1,43 @@
-import { Component } from '@angular/core';
+import { Component } from "@angular/core";
+import {
+  faPencil,
+  IconDefinition,
+  faTrashCan,
+} from "@fortawesome/free-solid-svg-icons";
+import { mockedCoursesList } from "./shared/mocks/mocks";
 
 @Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  selector: "app-root",
+  templateUrl: "./app.component.html",
+  styleUrls: ["./app.component.scss"],
 })
 export class AppComponent {
-  title = 'courses-app';
+  title = "courses-app";
+
+  //  text for buttons
+  btnTextLogin: string = "Login";
+  btnTextLogout: string = "Logout";
+  btnTextAddNewCourse: string = "Add New Course";
+  btnTextShowCourse: string = "Show Course";
+
+  // icons for buttons
+  faTrashCan: IconDefinition = faTrashCan;
+  faPencil: IconDefinition = faPencil;
+
+  // info page text
+  infoTitle = "Your List Is Empty";
+  infoText = "Please use 'Add New Course' button to add your first course";
+
+  // Mocked Course
+  mockedCourse = mockedCoursesList[0];
+  mockedCoursesList = mockedCoursesList;
+  showCourse(courseId: string) {
+    console.log("Clicked on Show Course with ID:", courseId);
```

Comment:
Just a note here, we do not usually leave `console.log`-s in our code as there are much better solutions available for logging and tracking.
You can leave this as is, just bear in mind that keep these to a minimum and only use when truly necessary.